src/services/legal-services.ts

```diff
@@ -38,7 +38,7 @@ export default class LegalServices {
    * Fetches the first or only filing.
    * This is probably a draft Amalgamation, IA or Registration.
    * @param tempId the temp registration number
-   * @returns a promise to return the draft filing, else exception
```

Comment:
I think we can generally assume that most methods can throw an exception, so no need to call it out.